doc/tests/manual/reports/230730.md

```diff
@@ -0,0 +1,318 @@
+# Manual Test Report (v0.3.1)
+
+## General
+
+- Tester: Aydan Namdar Ghazani
+- Date/Time: 30 July 2023
+- Duration: 10 minutes
+- Commit/Tag: [master eefcc45](https://github.com/ElektraInitiative/PermaplanT/tree/eefcc451c1a268cee28907864b7a60b4297dcc1d)
+- Setup: dev.permaplant.net
+- Planned tests: **14**
+- Executed tests: **14**
+- ✔️Passed tests: **14**
+- ⚠️Problematic tests: **0**
+- ❌Failed tests: **0**
+
+## [Error Analysis](../README.md#report-header)
+
+No errors occured.
+
+## Closing remarks
+
+### How is the current state of the software?
+
+The software is in a fully working state with no issues. Maybe the usability of the grid scale on a map could be improved.
```

Comment:
How should the grid be changed in your opinion?

Reply:
We fixed the problem in #744. The length indicator was in the way of the grid scale thingy.